practice-app/website/api/event.py

```diff
@@ -1,20 +1,24 @@
+from re import template
 from flask import Blueprint, request,  url_for, jsonify, make_response, abort, flash
-from ..models import Event
+from sqlalchemy.sql.elements import Null
+from ..models import Event, DiscussionPost
 from .. import db
+import json
+import sqlite3
 import requests
 from flasgger.utils import swag_from
 from sqlalchemy import exc
 events = Blueprint('events', __name__)
 
-API_KEY = '<api key>'
+API_KEY = 'AIzaSyB918Ru6ZsU1P3OPc-IaE_lRLgjNM1Suyk'
```

Comment:
this should not be put like this, we now use dotenv for this purpose.

Reply:
This issue does not exist after merging the master into this branch, and solved after resolving the conflicts.